rllib/policy/sample_batch.py

```diff
@@ -426,7 +426,7 @@ def size_bytes(self) -> int:
         Returns:
             int: The overall size in bytes of the data buffer (all columns).
         """
-        return sum(sys.getsizeof(d) for d in self.values())
+        return sum(d.nbytes for d in self.data.values())
```

Comment:
@sven1977 I think it's posible the values aren't numpy arrays right?

How about we use d.nbytes if isinstance(np.ndarray) otherwise, sys.getsizeof?

Reply:
If `d` isn't a numpy array, I think the best way is either 1) do nothing 2) throw a warning and do not estimate buffer size.

Actually `sys.getsizeof` is not guaranteed to get the real size of any non-primitive object.

In addition, as a user of rllib, I really suffered from the wrong estimation of memory usage. In my work, the estimation told me the buffer would take less than 1GB memory, but it actually would take dozens of GB. For this reason my program always went wrong and I saw some message like `the actor died unexpectedly`, which is difficult to locate the accurate bug.

Reply:
Correct, values dont have to be np arrays.
Can we add this check then? Also, please do not use `self.data` anymore. It's been deprecated as SampleBatches are now dicts.
So like this?
```
return sum(d.nbytes if isinstance(d, np.ndarray) else 0 for d in self.values())
```